control-plane/csi-driver/src/bin/node/nodeplugin_grpc.rs

```diff
@@ -46,18 +50,50 @@ impl NodePlugin for NodePluginSvc {
     ) -> Result<Response<FreezeFsReply>, Status> {
         let volume_id = request.into_inner().volume_id;
         debug!("freeze_fs({})", volume_id);
-        freeze_volume(&volume_id).await?;
-        Ok(Response::new(FreezeFsReply {}))
+        let _guard = VolumeFsfreezeGuard::new_str(&volume_id)?;
+        let output = Command::new("csi-node")
+            .arg("--node-name")
+            .arg("dummy")
```

Comment:
I think you can make node-name required only for when there's no subcommand?

Reply:
Done.